### ISSUE TITLE
dashboard not automatically refreshed

### ISSUE THREAD

**Issue by forrestbao** [codepod-io/codepod#383]
If I create a new repo, and click "Codepod" in the breadcrumbs to navigate back to the dashboard, the new repo will not appear on my dashboard until I manually fresh (via the browser) the dashboard page. I am not sure whether we do not have autorefresh or the auto-refresh frequency is too low. But at least every time I access it.

**Comment by lihebi** [comment]
Yes, I'm aware of this. This is a bug somewhere in React re-rendering logic. It should be instantly refreshed.

**Comment by sumtsui** [comment]
i can take a look

**Comment by senwang86** [comment]
PR #422 should fix the issue

**Comment by lihebi** [comment]
#422 doesn't fix this issue. (And #422 introduced a regression, fixed in #425).

This issue is weird, it works well (to refresh the dashboard) in the local development env, but doesn't work on the production server.

**Comment by lihebi** [comment]
Possible solutions:
1. use the full query `{query: ..., variables: ...}` instead of the string `"GetDashboardRepos"` in the `refetchQueries` option. Ref: https://github.com/apollographql/apollo-client/issues/5419#issuecomment-598065442
2. update GraphQL cache manually instead of relying on `refetchQueries`, similar to what Haochen did in https://github.com/codepod-io/codepod/pull/364#discussion_r1257722308
